### PR TITLE
Date corrected the example description from Sunday, to Friday, in the…

### LINKED ISSUE #39997: "Rosetta Code: Date format" sample day is incorrect
**Describe your problem and how to reproduce it:**

The sample output in the sidebar has the date `Sunday, November 23, 2007`.  The date should read `Friday, November 23, 2007`.  Source: https://howlongagogo.com/date/2007/november/23 and many other websites.

**Add a Link to the page with the problem:**

https://www.freecodecamp.org/learn/coding-interview-prep/rosetta-code/date-format

**Tell us about your browser and operating system:**
* Browser Name: Firefox
* Browser Version: 82
* Operating System: Linux Mint

### PR DESCRIPTION
**Pull request by SlickJacket**
… date-format Rosetta Code Challenge (English)

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39997 

If you could add the Hacktoberfest tag, that would mean the world! I learned a lot from you guys, and although this isn't the most impressive pull request, it is my first contribution to open source! Thank you guys again for everything

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on opening your first pull request! 😎 Please note that our workflow requires at least one additional approval before this can be merged.

**Review by majestic-owl448 — APPROVED** [review]
LGTM 👍

**Comment by Ieahleen** [comment]
Congratulations on your first merged pull request! 🎉 

If you want to continue contributing to free Code Camp take a look at the issues labelled as `help wanted`

**Pull request merged**
